### ISSUE TITLE
Ashen Armor

### ISSUE THREAD

**Issue by BlazeKing125** [EpicSquid/Embers#7]
The Ashen Armor, more specifically the chestplate, has a weird texture thing going with the arm parts, is it supposed to be this low, or is this a visual glitch? Thank you!!!
![2018-06-17_19 10 13](https://user-images.githubusercontent.com/33373922/41512984-45e4d6f4-7262-11e8-87e1-9f7ff28e6736.png)

**Comment by DaedalusGame** [comment]
hm, the code references those two bits as "gloves" so i believe they are supposed to be that low. is there anything else wrong with ashen armor? on the picture i see a slight desync between the armor and the player model but i don't get that in dev...

**Comment by Cronock92** [comment]
I'm having a problem with the armor as well but it seems to be all the armor pieces.
![2018-08-08_12 10 58](https://user-images.githubusercontent.com/42215379/43852935-37abfeb2-9b04-11e8-92af-8a0b76287d92.png)

**Comment by BlazeKing125** [comment]
Is that mod with mo-bends, if so mo-bends doesnt have any cross compatibility with custom armor models atm.

**Comment by NovaLynxie** [comment]
Yeah, I've seen this same glitch with different mods as well, not just Embers. Is there any plans to make this work properly with custom armor models later on as I really love this mod.
